src/ert/config/analysis_config.py

```diff
@@ -181,10 +181,16 @@ def from_dict(cls, config_dict: ConfigDict) -> AnalysisConfig:
             for design_matrix_config_list in design_matrix_config_lists
         ]
         design_matrix: DesignMatrix | None = None
+
         if design_matrices:
```

Comment:
Can you add a general logging when somebody actually uses this keyword?

Reply:
yes we do. @sondreso asked if we can log it explicitly but I guess having the full config should be ok.